lib/ansible/modules/cloud/amazon/glue_job.py

```diff
@@ -0,0 +1,377 @@
+#!/usr/bin/python
```

Comment:
I think `aws_glue_job` would be a more appropriate name - because isn't most technology basically glue? :wink: